plugins/azure/virtualmachines/vmDiskPublicAccess.js

```diff
@@ -0,0 +1,57 @@
+var async = require('async');
+
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'VM Disk Public Access',
+    category: 'Virtual Machines',
+    domain: 'Compute',
+    description: 'Ensures that Azure virtual machine disks are not accessible publicly.',
+    more_info: 'Private endpoints restrict the export and import of managed disks and more securely access data over a private link from clients on your Azure virtual network. The private endpoint uses an IP address from the virtual network address space for your managed disks. Network traffic between clients on their virtual network and managed disks only traverses over the virtual network and a private link on the Microsoft backbone network, eliminating exposure from the public internet.',
```

Comment:
```suggestion
    more_info: 'Private endpoints safeguard against unauthorized access and cyber threats, preserving the integrity and confidentiality of your data while aligning with compliance and security best practices by restricting the export and import of managed disks and only allowing access over a private link from clients on your Azure virtual network.',
```
make it a little shorter